### PR TITLE
Document specifications we aim to support

### LINKED ISSUE #124: User/Priv spec
I see that the current code conforms to PRIV 1.11, But I cannot find anywhere which indicates the USER level, I would presume 2.3, but I guess it could be 2.2. In which case where should I look I cannot find it in the docs
Thx
Lee

### PR DESCRIPTION
**Pull request by imphil**
Fixes #124

### CODE REVIEW

File: doc/introduction.rst [hunk 1]
```diff
@@ -11,17 +11,52 @@ Ibex has been designed to be small and efficient.
...
+* The RV32E Base Integer Instruction Set, version 1.9 (draft from June 8, 2019)
+
+In addition, the following instruction set extensions are available.
+
+.. list-table:: Ibex Instruction Set Extensions
+   :header-rows: 1
+
+   * - Extension
+     - Version
+     - Optional?
```
Comment (vogelpi) [inline doc/introduction.rst]: Configurability

File: doc/introduction.rst [hunk 2]
```diff
@@ -11,17 +11,52 @@ Ibex has been designed to be small and efficient.
...
+
+   * - C: Standard Extension for Compressed Instructions
```
Comment (vogelpi) [inline doc/introduction.rst]: Make "C" bold.

File: doc/introduction.rst [hunk 3]
```diff
@@ -11,17 +11,52 @@ Ibex has been designed to be small and efficient.
...
+     - 2.0
+     - always enabled
+
+   * - M: Standard Extension for Integer Multiplication and Division
```
Comment (vogelpi) [inline doc/introduction.rst]: Make "M" bold.

File: doc/introduction.rst [hunk 4]
```diff
@@ -11,17 +11,52 @@ Ibex has been designed to be small and efficient.
...
+     - 2.0
+     - optional
+
+   * - Zicsr: Control and Status Register Instructions
```
Comment (vogelpi) [inline doc/introduction.rst]: Make "Zicsr" bold.

File: doc/introduction.rst [hunk 5]
```diff
@@ -11,17 +11,52 @@ Ibex has been designed to be small and efficient.
...
+     - 2.0
+     - always enabled
+
+Most content of the RISC-V privileged specification is optional.
+Ibex currently supports the following features.
```
Comment (vogelpi) [inline doc/introduction.rst]: Maybe add "...according to version 1.11 of the spec." just to list the spec version as for the user-level ISA above.

**Comment by imphil** [comment]
Thanks @vogelpi! Updated and merged.

**Pull request merged**